src/panels/lovelace/cards/hui-glance-card.js

```diff
@@ -55,36 +56,67 @@ class HuiGlanceCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
         .name {
           min-height: var(--paper-font-body1_-_line-height, 20px);
         }
+        state-badge {
+          margin: 8px 0;
+        }
       </style>
+    `;
+  }
+
+  renderEntity(entityConf) {
+    const stateObj = this.hass.states[entityConf.entity];
 
-      <ha-card header="[[_config.title]]">
-        <div class$="[[_computeClasses(_config.title)]]">
-          <template is="dom-repeat" items="[[_configEntities]]">
-            <template is="dom-if" if="[[_showEntity(item, hass.states)]]">
-              <div class="entity" on-click="_handleClick">
-                <template is="dom-if" if="[[_showInfo(_config.show_name)]]">
-                  <div class="name">[[_computeName(item, hass.states)]]</div>
-                </template>
-                <state-badge
-                  state-obj="[[_computeStateObj(item, hass.states)]]"
-                  override-icon="[[item.icon]]"
-                ></state-badge>
-                <template is="dom-if" if="[[_showInfo(_config.show_state)]]">
-                  <div>[[_computeState(item, hass.states)]]</div>
-                </template>
-              </div>
-            </template>
-          </template>
+    return html`
+      <div class="entity" .entityConf="${entityConf}" @click="${
+      this._handleClick
+    }">
```

Comment:
These two lines look out of place. Maybe
```html
<div 
  class="entity"
  .entityConf="${entityConf}"
  @click="${this._handleClick}">
```

Reply:
This will be better once prettier supports HTML. Until then, this is how it's going to be as we don't control the styling anymore.

Reply:
Ha, no realized I can format the HTML actually. done